udata_front/theme/gouvfr/assets/js/components/AdminTable/AdminDatasetsTable/AdminDatasetsTable.vue

```diff
@@ -0,0 +1,191 @@
+<template>
+  <AdminTable>
+    <thead>
+      <tr>
+        <AdminTableTh
+          :sortable="true"
+          @sort="(direction) => updateSort('title', direction)"
+          :sorted="sorted('title')"
+          scope="col"
+        >
+          {{ t('Title of the dataset') }}
+        </AdminTableTh>
+        <AdminTableTh scope="col">{{ t("Status") }}</AdminTableTh>
+        <AdminTableTh
+          :sortable="true"
+          @sort="(direction) => updateSort('created', direction)"
+          :sorted="sorted('created')"
+          scope="col"
+        >
+          {{ t('Created at') }}
+        </AdminTableTh>
+        <AdminTableTh
+          :sortable="true"
+          @sort="(direction) => updateSort('last_update', direction)"
+          :sorted="sorted('last_update')"
+          scope="col"
+        >
+          {{ t('Updated at') }}
+        </AdminTableTh>
+        <AdminTableTh scope="col">{{ t('Files') }}</AdminTableTh>
+        <AdminTableTh scope="col">{{ t('Score') }}</AdminTableTh>
+        <AdminTableTh scope="col">
+          <Tooltip>
+            <span class="fr-icon--sm fr-icon-chat-3-line" aria-hidden="true"></span>
+            <template #tooltip> {{ t('Discussions') }}</template>
+          </Tooltip>
+        </AdminTableTh>
+        <AdminTableTh
+          :sortable="true"
+          @sort="(direction) => updateSort('views', direction)"
```

Comment:
Should we sort by `-views` on first click instead of ascending on the metrics columns?
The sorting order would then differ between columns though.
In current admin, seems like we're always sorting in descending order (title, dates, metrics).
cc @agarrone

Reply:
done !